guide/portuguese/react/react-router/index.md

```diff
@@ -8,17 +8,17 @@ localeTitle: React Router
 
 O React Router foi dividido em três pacotes: `react-router` , `react-router-dom` e `react-router-native` .
 
-Você quase nunca precisa instalar diretamente o roteador reage. Esse pacote fornece os principais componentes e funções de roteamento para os aplicativos do React Router. Os outros dois fornecem componentes específicos do ambiente (navegador e reagir-nativo), mas ambos também reexportam todas as exportações do roteador-reagente.
+Você quase nunca precisará instalar diretamente o react-router. Esse pacote fornece os principais componentes e funções de roteamento para os aplicativos do React Router. Os outros dois fornecem componentes específicos do ambiente (navegador e reagir-nativo), mas ambos também reexportam todas as exportações do react-router.
```

Comment:
I suggest you use markdown in the words `react-router` , `react-router-dom` e `react-router-native`. Where is " (navegador e reagir-nativo)" I suggest you change to "(navegador e react-native)".

```suggestion
Você quase nunca precisará instalar diretamente o react-router. Esse pacote fornece os principais componentes e funções de roteamento para os aplicativos do React Router. Os outros dois fornecem componentes específicos do ambiente (navegador e reagir-nativo), mas ambos também reexportam todas as exportações do react-router.
```

Reply:
@igorhaugg Please make the changed suggested by @CleoAguiar, so we can merge this PR.

Thank you,.